### PR TITLE
Update project profile 3818

### LINKED ISSUE #3818: Update Project Profile: TDM Calculator
### Overview
As a developer, we need to ensure our project's details are up to date on the HackforLA website. We need to update the "Current Project Team" for the [TDM Calculator Project Page](https://www.hackforla.org/projects/tdm-calculator).

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of the [tdm-calculator.md](https://github.com/hackforla/website/blob/gh-pages/_projects/tdm-calculator.md) file and find the `leadership` variable.
- [x] Remove Renee Moore-Adams from the `leadership` variable
- [x] Add the new team members names, roles, slack links, github links, and picture to the end of the `leadership` variable
    - [x] Name: Anousha Shadrach
GitHub Handle: Noushie
Role: UX Designer
Slack Member ID: U045EHW8NDA
    - [x] Name: Moji Momoh
GitHub Handle: mojimoh
Role: Product Manager
Slack Member ID: D04HDM7DM18

- [x] Check the [TDM Calculator Project Page](http://localhost:4000/projects/tdm-calculator) on your local development environment to ensure the "Current Project Team" section is properly displaying the changes you made

### Instructions
- For the github link, append the team member's github handle to https://github.com/
- For the picture link, append the team member's github handle to https://avatars.githubusercontent.com/
- For the slack link, append the team member's slack ID to https://hackforla.slack.com/team/

### Resources
- [tdm-calculator.md](https://github.com/hackforla/website/blob/gh-pages/_projects/tdm-calculator.md)
- [HFLA wiki on project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)
- [How to find Slack IDs](https://github.com/hackforla/admin/wiki/Slack:-Find-a-Slack-ID)

### PR DESCRIPTION
**Pull request by kotynskm**
Fixes #3818

### What changes did you make and why did you make them ?

  - Removed Renee Moore-Adams from leadership
  - Added Anousha Shadrach and Moji Momoh to leadership
  - Updated links for each new leadership member

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

Screenshot 2023-01-31 at 4 11 22 PM

<summary>Visuals after changes are applied</summary>
  
Screenshot 2023-01-31 at 3 39 10 PM

</details>

### CODE REVIEW

**Comment by kotynskm** [comment]
I forgot to take a screenshot of before changes, I tried to revert to the original, but then the site did not update for me. I put back the correct changes and now I don't see any content for the tdm project page. Please let me know if there are any issues.

**Comment by vmaineng** [comment]
ETA: End of the day on 2/1
Availability: 2 hours

**Comment by vmaineng** [comment]
After reviewing this PR, it looks good to be merged in and met all the requirements.

**Review by vmaineng — DISMISSED** [review]
After reviewing this PR, it looks good to be merged in and met all the requirements.

**Comment by MattPereira** [comment]
**ETA:** 20 minutes
**Availability:** 8 - 10 PM Sunday - Thursday

File: _projects/tdm-calculator.md [hunk 1]
```diff
@@ -24,12 +24,18 @@ leadership:
       slack: 'https://hackforla.slack.com/team/U031EU2E6LB'
       github: 'https://github.com/Biuwa'
     picture: https://avatars.githubusercontent.com/Biuwa
-  - name: Renee Moore-Adams
-    role: Lead, UX Design
+   - name: Anousha Shadrach
```
Comment (MattPereira) [inline _projects/tdm-calculator.md]: Hey @kotynskm it looks like you accidentally used an extra space on `line 27` before the `- name:` which is causing the docker container to throw a YAML Error and the tdm-calculator project information to not display across the website. YAML is extremely picky about indentation lol

Everything else looks great ! :+1: 

YAML_error

Screenshot from 2023-02-02 11-06-23

**Review by MattPereira — CHANGES_REQUESTED** [review]

File: _projects/tdm-calculator.md (hunk 1 above)
Comment (kotynskm) [reply _projects/tdm-calculator.md]: Thanks for pointing this out! I believe I found it and fixed the issue. 👍🏻

**Review by MattPereira — APPROVED** [review]
Solid work on this one @kotynskm ! :+1: 

Appreciate your timely response to my requested changes. The LA TDM Calculator details are now up to date well done!

**Pull request merged**